IdentityService/API/Jwt.cs

```diff
@@ -0,0 +1,13 @@
+﻿public static class Jwt
+{
+    public static SymmetricSecurityKey Key = new("TESTTESTTESTTEST"u8.ToArray());
```

Comment:
The key should not be hardcoded, it should be taken from some secure storage, or at least from appsetting

Reply:
Fixed